scripts/import/import_all_sv_data.pl

```diff
@@ -428,62 +427,45 @@ sub study_table{
       $study_desc = $1;
     }
 
-    my $external_link_sql;
-    if ($external_link eq 'NULL') {
-      if ($study_xref && $study_xref !~ /NULL/i && $study_xref ne '') {
-        $external_link_sql = '';
-      }
-      else {
-        $external_link_sql = "external_reference='$external_link',";
-      }
-    }
-    else {
-      $external_link_sql = "external_reference='$external_link',";
+    if ($external_link && $external_link eq 'NULL' && $external_link eq '') {
+      $external_link = undef;
     }
 
-    $stmt = qq{ UPDATE $study_table SET 
-                  description='$study_desc',
-                  $external_link_sql
-                  study_type="$study_type",
-                  url="$study_ftp"
-                WHERE study_id=$study_id
-              };
+    $stmt = $dbVar->prepare(qq[ UPDATE $study_table SET description = ?, external_reference = ?, study_type = ?, url = ? WHERE study_id = ? ]);
+    $stmt->execute($study_desc,
+                  $external_link || undef,
+                  $study_type || undef,
+                  $study_ftp || undef,
+                  $study_id
+                  );
 
-    $dbVar->do($stmt);
   }
   # INSERT
   else {
-    if ($external_link && $external_link ne 'NULL' && $external_link ne '') {
-      $external_link = "'$external_link'";
+    if($external_link && $external_link eq 'NULL') {
+      $external_link = undef;
```

Comment:
this is similar to L430 - can nulls be dealt with once per input line?

Reply:
Good point. I'll do this before the if .. else